packages/@lwc/integration-tests/scripts/build.js

```diff
@@ -112,7 +111,10 @@ testEntries
             templates.html(testName),
             'utf8'
         );
-    }, Promise.resolve())
-    .catch((err) => {
-        console.log(err);
-    });
+    }
+}
```

Comment:
I noticed some errors in CircleCI; this should fix it by 1) not using `fs-extras`, which isn't even a direct dependency of ours, and 2) doing all the builds/writes serially.

```
(node:2027) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 19 error listeners added to [WriteStream]. Use emitter.setMaxListeners() to increase limit

```